.github/actions/write-config/action.yml

```diff
@@ -0,0 +1,43 @@
+name: 'write-config'
```

Comment:
maybe name it `populate-test-credentials` or something. Write config seems too generic.